Firebase/Auth/Source/AuthProviders/Phone/FIRPhoneAuthProvider.m

```diff
@@ -403,12 +409,16 @@ - (void)reCAPTCHAURLWithCompletion:(FIRReCAPTCHAURLCallBack)completion {
     NSString *bundleID = [NSBundle mainBundle].bundleIdentifier;
     NSString *clienID = _auth.app.options.clientID;
     NSString *apiKey = _auth.app.options.APIKey;
+    _eventID = [NSString stringWithFormat:@"%c%c", 64 + arc4random_uniform(26), 64 +
```

Comment:
While we're at it, let's make it a little longer. I don't think a small loop is that bad. Also, please use character literal ('a' and 'z' - 'a') instead of ASCII code (64 and 26).

Reply:
Done thanks.